src/main/resources/lib/configFile/configFile.js

```diff
@@ -0,0 +1,180 @@
+const AUTOINIT="autoinit"
+
+// Expected format in this app's .cfg file, for separately configuring multiple idproviders: idprovider.<name>.<field>[.optional][.subfields][.etc...]
+// For example, an idprovider named "myidp", with a configuration tree { mySetting1: "false", mySetting2: { nextLevel: true } }:
+//
+// idprovider.myidp.mySetting1=false
+// idprovider.myidp.mySetting2.nextLevel=true
+//
+const CONFIG_NAMESPACE = "idprovider";
+exports.CONFIG_NAMESPACE = CONFIG_NAMESPACE;
+
+
+
+// TODO: Redundant? Duplicates appear to be silently skipped when .cfg is read?
```

Comment:
`.cfg` is `.properties` file. Which is in turn converted to Map. There will be no visible duplicates
BTW, that Map is unsorted.

Reply:
Function removed.